### DISCUSSION TITLE
Can not access the repository for Cent Os installation

### DISCUSSION THREAD

**Discussion by Babakjfard** [GeoDaCenter/geoda#2384] [Q&A]
Hello,
I am trying to install GeoDa on a CentOs system, following the instructions in this link:
https://github.com/GeoDaCenter/geoda/tree/master/BuildTools/centos

But the repository needs username and password to access. Is there anyway that I can set it up?

yours,
 Babak

**Comment by jkoschinsky** [comment]
This is an old set of instructions pointing to an unused repo:

Use SVN to check out GeoDa trunk:
 - From user's home directory: ~/
 - svn co https://geodacenter.repositoryhosting.com/svn/geodacenter_geoda/trunk trunk

We didn’t support CentOS binary for a while. But you should be able to follow the Ubuntu installation to create one for CentOS.

Julia Koschinsky, Ph.D.
Executive Director and Senior Research Associate
Center for Spatial Data Science
University of Chicago
https://spatial.uchicago.edu/

<https://spatial-data-science-conference.com/>
________________________________
From: Babak Fard ***@***.***>
Sent: Wednesday, April 20, 2022 5:39 PM
To: GeoDaCenter/geoda ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [GeoDaCenter/geoda] Can not access the repository for Cent Os installation (Discussion #2384)


Hello,
I am trying to install GeoDa on a CentOs system, following the instructions in this link:
https://github.com/GeoDaCenter/geoda/tree/master/BuildTools/centos

But the repository needs username and password to access. Is there anyway that I can set it up?

yours,
Babak
